gen-models/src/accession.rs

```diff
@@ -271,11 +273,12 @@ impl Accession {
     }
 
     pub fn get_or_create(
-        conn: &Connection,
+        conn: &GraphConnection,
```

Comment:
these prevent us from passing OperationConnection, so we don't get mixups anymore